### PR TITLE
Fix pl-threejs element

### PR DESCRIPTION
**Pull request by nwalters512**
This element has been broken in a few ways for a while now:

- The entire element didn't render because files were moved around in the `three` package.
- Once that was fixed, the motion toggles didn't work, and the motion buttons displayed a missing icon error.

This PR fixes both these issues. It also adds new tests for element assets to prevent future regressions.

### CODE REVIEW

File: elements/pl-threejs/info.json [hunk 1]
```diff
@@ -3,15 +3,15 @@
     "dependencies": {
         "nodeModulesScripts": [
             "three/build/three.min.js",
```
Comment (nwalters512) [inline elements/pl-threejs/info.json:5]: We're getting a warning that they'll be removing the pre-built threejs files soon in favor of ESM modules. This makes sense from a moving-the-ecosystem-forward perspective, but is going to cause us some pain 😢

**Comment by jonatanschroeder** [comment]
Is it worth it to handle #5540 here?

**Comment by nwalters512** [comment]
IMO no; I'm trying to do the smallest amount of work possible to proceed with https://github.com/PrairieLearn/PrairieLearn/pull/7582. This PR only aims to return the element to a working state, not make any other changes to it.

File: elements/pl-threejs/OrbitControls.js
Comment (mwest1066) [inline elements/pl-threejs/OrbitControls.js:1]: So nasty to include these in our our source tree, but I agree it's the best bad option at the moment.

**Review by mwest1066 — APPROVED** [review]

File: elements/pl-threejs/OrbitControls.js
Comment (nwalters512) [reply elements/pl-threejs/OrbitControls.js:1]: Yeah. Once Three forces our hand into consuming ESM, we can consider some little `@prairielearn/three` package that bundles Three + these other files into a single file that we can use in the element.

**Pull request merged**